workshops/geometric_pattern/README.md

```diff
@@ -1,15 +1,22 @@
 # Geometric Pattern
 
-![](img/sample1.png)
+| What you'll build    |
+| -------------------- |
+| ![](img/sample1.png) |
 
-Short link to this workshop: https://workshops.hackclub.com/geometric_pattern
+Here's a link to the [live demo][live_demo] and the [final code][final_code]. This workshop should take around 1 hour.
 
-[Demo](http://prophetorpheus.github.io/geometric_pattern/)
+_**Warning: this demo involves flashing colors and is not recommended for those with a history of epilepsy.**_ There's also an [animated demo][animated_live_demo].
```

Comment:
> this demo

I'd change this to "the following", as I'm /pretty/ sure that that makes more sense contextually. But like, that's really minor...